kubernetes/manifests.go

```diff
@@ -295,7 +295,7 @@ data:
           fallthrough in-addr.arpa ip6.arpa
         }
         prometheus :9153
-        forward . /etc/resolv.conf
+        forward . 8.8.8.8
```

Comment:
@dontlaugh for the record, this is why this PR is WIP - I hacked this on because I cannot for the life of me figure out why I am getting some kind of DNS loop error. The local nameserver works just fine, but for whatever reason trying to access anything outside of the cluster isn't working.

Reply:
We can expect LXD servers to have opinions on this, and trying to set this up in an agnostic way is part of the reason I have misgivings about creating our own network types on the fly.

Reasons I've seen for DNS failure:
* DHCP fails and container is not given an IP (This happens mostly with non-ubuntu images, though)
* The underlying _host_ does not have IPv4 forwarding set up, or otherwise the container subnet (usually attached to a bridge named `lxdbr0`) has no route off the host

Reply:
To be clear, I'm not against setting up our own network, but I don't think it will be simple to keep it robust.